lib/user.rb

```diff
@@ -0,0 +1,41 @@
+class User < Sequel::Model(:userdetails)
+  User.unrestrict_primary_key
```

Comment:
What's this all about?

Reply:
because the username is the primary key for the table it can't be mass assigned in a `.create` unless we do this

Reply:
What do you mean by mass assigned?  Where do we do that in the code?

Reply:
`User.create(username: random_username)...` is mass assignment

Reply:
https://github.com/jeremyevans/sequel/blob/master/doc/mass_assignment.rdoc